### PR TITLE
Some improvements to e2e tests

### PR DESCRIPTION
**Pull request by krzkaczor**
- [x] make all tests pass locally
- [x] make screenshots deterministic

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -39,25 +39,38 @@ jobs:
...
-            docker run -d -p 8545:8545 oasisdex/localnode:$LOCALNODE_SHA master
-      - run: 
+            docker run -d -p 8545:8545 oasisdexorg/localnode:$LOCALNODE_SHA master
+      - run:
           name: Wait for localnode to be up
           command: |
             ./.circleci/wait-for-it.sh localhost:8545 -t 10
             sleep 10
-      - run: 
+      - run:
```
Comment (peculiarity) [inline .circleci/config.yml]: I have problems with my eyes or this is duplicate

Comment (krzkaczor) [reply .circleci/config.yml]: i know right! it's just a to stress test e2e setup. find tests that are failing etc

Comment (peculiarity) [reply .circleci/config.yml]: so you want to have 2 setups running  :)  I feel you

Comment (krzkaczor) [reply .circleci/config.yml]: copying this few times was much faster than learning how to do proper loops in bash 😆

**Pull request merged**